tests/cases/conformance/jsdoc/jsdocAbstract4.ts

```diff
@@ -0,0 +1,11 @@
+// @allowJs: true
+// @checkJs: true
+// @target: esnext
+// @declaration: true
+// @emitDeclarationOnly: true
+// @Filename: jsdocAbstract4.js
+
+class Foo {
+    /** @abstract */
+    foo() {}
```

Comment:
consider adding a test with a static abstract method to ensure there is an error either on `static` or `@abstract`